uint16 planeID

float64 scurve

uint8 mode

uint8 FW_IN_IDEL = 0 

uint8 FW_IN_TAKEOFF = 1

uint8 FW_IN_LANDING = 2

uint8 FW_IN_FORMATION = 3

uint8 FW_IN_PROTECT = 4
